src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelSchema.scala

```diff
@@ -59,3 +58,42 @@ case class EntityMetadata(
 )
 
 case class EntityModel(schema : Map[String, EntityMetadata]) //entity name -> stuff about it
+
+object FlexibleModelSchema {
```

Comment:
does it make sense for FlexibleModelSchema to be an object? i think after i implement the upload, we may be able to get rid of ModelSchema.